### PR TITLE
zh-cn(update): sync connet-src for the English

### PR DESCRIPTION
**Pull request by fwqaaq**
# partof #3893

### CODE REVIEW

File: files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md [hunk 1]
```diff
@@ -5,77 +5,97 @@ slug: Web/HTTP/Headers/Content-Security-Policy/connect-src
...
-## Syntax
-
-connect-src 可以设置一个或者多个源地址：
-
-```plain
+- {{domxref("WebSocket")}},
+- {{domxref("EventSource")}}, and
+- {{domxref("Navigator.sendBeacon()")}}.
+
+> **备注：** `connect-src 'self'` 并不是所有浏览器都能解析为 websocket 协议，更多信息，请查看这个 [issue](https://github.com/w3c/webappsec-csp/issues/7)。
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md:17]:
```suggestion
> **备注：** 并不是所有浏览器都能将 `connect-src 'self'` 解析为 websocket 协议，更多信息，请查看这个 [issue](https://github.com/w3c/webappsec-csp/issues/7)。
```
```suggestion
> **备注：** `connect-src 'self'` 并不是所有浏览器都能解析为 websocket 协议，更多信息，请查看这个 [issue](https://github.com/w3c/webappsec-csp/issues/7)。
```

File: files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md [hunk 2]
```diff
@@ -5,77 +5,97 @@ slug: Web/HTTP/Headers/Content-Security-Policy/connect-src
...
-给定如下 CSP 头部：
+给定此 CSP 标头：
 
-```bash
+```http
 Content-Security-Policy: connect-src https://example.com/
 ```
 
-如下的连接请求会被阻塞且不会加载：
+一下连接被禁止并且将不会夹在：
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md:63]:
```suggestion
以下连接被禁止并且将不会加载：
```

File: files/zh-cn/web/http/headers/content-security-policy/font-src/index.md [hunk 3]
```diff
@@ -57,7 +73,7 @@ Content-Security-Policy: font-src https://example.com/
 
 {{Compat}}
 
-## See also
+## 参见
 
-- {{HTTPHeader("Content-Security-Policy")}}
+- {{HTTPHeader("Content-Security-Policy")}}（内容安全性策略）
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/font-src/index.md:78]:
```suggestion
- {{HTTPHeader("Content-Security-Policy")}}（内容安全策略）
```

File: files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md [hunk 4]
```diff
@@ -5,77 +5,97 @@ slug: Web/HTTP/Headers/Content-Security-Policy/connect-src
...
-| {{CSP("default-src")}} fallback | Yes. If this directive is absent, the user agent will look for the `default-src` directive. |
-
-## Syntax
-
-connect-src 可以设置一个或者多个源地址：
-
-```plain
+- {{domxref("WebSocket")}},
+- {{domxref("EventSource")}}, and
+- {{domxref("Navigator.sendBeacon()")}}.
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md:15]: 可以移除这里的逗号和 `and`

File: files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md [hunk 5]
```diff
@@ -5,77 +5,97 @@ slug: Web/HTTP/Headers/Content-Security-Policy/connect-src
...
+      <td>1</td>
+    </tr>
+    <tr>
+      <th scope="row"> 指令类型</th>
+      <td>{{Glossary("Fetch directive","Fetch 指令")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">{{CSP("default-src")}} 回落</th>
+      <td>
+        是的。如果没有此指令，用户代理将查找 <code>default-src</code> 指令。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md:32]:
```suggestion
        存在。如果没有此指令，用户代理将查找 <code>default-src</code> 指令。
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md (hunk 4 above)
Comment (fwqaaq) [reply files/zh-cn/web/http/headers/content-security-policy/connect-src/index.md]:
```suggestion
- {{domxref("WebSocket")}}
- {{domxref("EventSource")}}
- {{domxref("Navigator.sendBeacon()")}}
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**